docs/installation.rst

```diff
@@ -4,11 +4,76 @@
 Installation
 ============
 
-At the command line type::
+As mentioned in the :doc:`index`, HistomicsTK can be used both as a pure
```

Comment:
This resolves in readthedocs as "As mentioned in the HistomicsTK, HistomicsTK can ...".  I think you need ``:doc:`index <index>```.